templates/github/.github/workflows/create-branch.yml.j2

```diff
@@ -23,9 +23,11 @@ jobs:
     steps:
       {{ checkout(depth=0, path=plugin_name) | indent(6) }}
 
+      {{ checkout(repository="pulp/plugin_template", path="plugin_template") | indent(6) }}
+
       {{ setup_python() | indent(6) }}
 
-      {{ install_python_deps(["bump2version", "jinja2", "pyyaml", "packaging", "requests"]) | indent(6) }}
+      {{ install_python_deps(["bump2version", "jinja2", "pyyaml", "packaging", "-r", "plugin_template/requirements.txt"]) | indent(6) }}
```

Comment:
I think jinja2 and pyyaml need to be included in the requirements file: https://github.com/pulp/plugin_template/blob/c66a678a4c2685c4589e5e94591485cb02b5e657/plugin-template#L14-L15